lib/reporter.js

```diff
@@ -0,0 +1,157 @@
+const metrics = require('wavefrontmetrics');
+const config = require('./config')
```

Comment:
Be consistent with semi-colons.  Recommend you run this through a linter before committing.